### PR TITLE
Build packages for linux/arm64

### PR DESCRIPTION
**Pull request by rongxin-liu**
This PR adds support to build packages for the linux/arm64 platform.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,29 +3,45 @@ jobs:
...
-    - name: Build
-      run: |
-        make deb
-        make rpm
-    - name: Deploy
-      if: ${{ github.ref == 'refs/heads/main' }}
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
```
Comment (dmalan) [inline .github/workflows/main.yml:7]: Can we (easily) avoid using a third-party action?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -3,29 +3,45 @@ jobs:
...
-        sudo gem install fpm package_cloud
-    - name: Build
-      run: |
-        make deb
-        make rpm
-    - name: Deploy
-      if: ${{ github.ref == 'refs/heads/main' }}
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
```
Comment (dmalan) [inline .github/workflows/main.yml:6]: Do we need to fixate on 2.1.0 or does v2 suffice? Or v2.1?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -3,29 +3,45 @@ jobs:
...
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build packages for aarch64 platform
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          dockerRunArgs: |
+            --volume ${PWD}/artifacts:/artifacts
+          run: |
+            apt-get update
+            apt-get install -y rpm gem ruby-full build-essential
```
Comment (dmalan) [inline .github/workflows/main.yml:16]: Let's alphabetize package names for readability?

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -3,29 +3,45 @@ jobs:
...
-      run: |
-        PACKAGECLOUD_REPO="cs50/repo"
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build packages for aarch64 platform
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          dockerRunArgs: |
+            --volume ${PWD}/artifacts:/artifacts
```
Comment (dmalan) [inline .github/workflows/main.yml:13]: Is `artifacts/` a convention or should we use, e.g., `build/`?

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -3,29 +3,45 @@ jobs:
...
+        run: |
+          PACKAGECLOUD_REPO="cs50/repo"
 
-        # Deploy deb to ubuntu repos
-        UBUNTU_REPOS=( xenial yakkety zesty artful bionic disco eoan focal groovy )
-        for repo in "${UBUNTU_REPOS[@]}"; do
-          package_cloud push "$PACKAGECLOUD_REPO"/ubuntu/"$repo" build/deb/*.deb
-        done
+          # Deploy deb to ubuntu repos
+          UBUNTU_REPOS=( xenial yakkety zesty artful bionic disco eoan focal groovy )
```
Comment (dmalan) [inline .github/workflows/main.yml:37]: Are these sorted from oldest to newest?

File: .github/workflows/main.yml (hunk 4 above)
Comment (rongxin-liu) [reply .github/workflows/main.yml:13]: `artifacts` is the folder specified by GitHub action, so I think we might want to keep this naming convention.

File: .github/workflows/main.yml (hunk 5 above)
Comment (rongxin-liu) [reply .github/workflows/main.yml:37]: Yes, according to the yank string listed [here](https://packagecloud.io/docs)

File: .github/workflows/main.yml (hunk 2 above)
Comment (rongxin-liu) [reply .github/workflows/main.yml]: Nope, v2 will suffice, updated.

**Review by dmalan — APPROVED** [review]

**Pull request merged**